tests/rptest/tests/cluster_features_test.py

```diff
@@ -27,7 +27,7 @@
 # The upgrade tests defined below rely on having a logical version lower than
 # CURRENT_LOGICAL_VERSION. For the sake of these tests, the exact version
 # shouldn't matter.
-OLD_VERSION = (22, 1, 4)
```

Comment:
For this test, we should probably just use the previous version (relative to head), as it's testing the feature manager in general, rather than any particular feature added in 22.2.

We can pick that up in a followup though: this patch is still a net improvement to make the test not skip versions.

Reply:
previous feature release, as in v23.1.x -> v22.3.x? I can do it now like I did (commits incoming soon) for partition_movement_upgrade_test